sonic_platform_base/sonic_storage/storage_devices.py

```diff
@@ -0,0 +1,89 @@
+#
+# storage_devices.py
+#
+# Composition class for the instantiation of objects of various storage device classes,
+# i.e. SsdUtil, EmmcUtil
+#
+
+
+try:
+    import os
+    import sys
+    from sonic_py_common import syslogger
+    from sonic_platform_base.sonic_storage.ssd import SsdUtil
+    from sonic_platform_base.sonic_storage.emmc import EmmcUtil
+except ImportError as e:
+    raise ImportError (str(e) + "- required module not found")
+
+# Not currently supported
+try:
+    from sonic_platform_base.sonic_storage.usb import UsbUtil
+except ImportError as e:
+    pass
+
+log_identifier = "StorageDevices"
+log = syslogger.SysLogger(log_identifier)
```

Comment:
@assrinivasan can you add this in __init__()

Reply:
Done in latest